cluster-autoscaler/cloudprovider/alicloud/alibaba-cloud-sdk-go/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
We have Apache license in the top level directory. Please do not provide additional licenses around.